.storybook/preview.js

```diff
@@ -1,5 +1,10 @@
 export const parameters = {
-  actions: { argTypesRegex: "^on[A-Z].*" },
+  actions: { 
+    argTypesRegex: "^on[A-Z].*",
+    argTypesRegex: "^fetch[A-Z].*" ,
+    argTypesRegex: "^create[A-Z].*",
```

Comment:
Correct me if I'm misunderstanding but aren't these key pairs useless here?

Reply:
> Correct me if I'm misunderstanding but aren't these key pairs useless here?

yeah, I was trying some fixes in regards with the action tab not firing on storybook, though it is now fixed.
I will remove these on later commits.